### PR TITLE
upgrade versions and warnings

### PR DESCRIPTION
**Pull request by tb102122**
*Issue #, if available:*  https://github.com/aws-samples/amazon-textract-response-parser/issues/101

*Description of changes:* upgrade modules to latest versions

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by tb102122** [comment]
@Belval @schadem when do you think can we approve that upgrade? I also still get an error message for dependency conflicts for amazon-textract-textractor 1.0.23 requires amazon-textract-response-parser==0.1.37, but you have amazon-textract-response-parser 0.1.42 which is incompatible.

File: helper/setup.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def _copy_fonts(self):
 
 requirements = [
     'boto3', 'botocore', 'amazon-textract-response-parser>=0.1.40', 'amazon-textract-caller>=0.0.27',
-    'amazon-textract-overlayer>=0.0.10', 'amazon-textract-prettyprinter>=0.1.0', 'Pillow>=9.2.*', 'PyPDF2>=2.5.*'
```
Comment (schadem) [inline helper/setup.py:55]: Why change PyPDF2

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
       # Checks out the repository
       - uses: actions/checkout@v3
         with:
-          ref: 'master'
+          ref: "master"
 
       # Upgrade pip
       - name: Upgrade pip
```
Comment (schadem) [inline .github/workflows/release.yml:22]: Why the changes in this file?

File: overlayer/setup.py [hunk 3]
```diff
@@ -7,7 +7,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-requirements = ['boto3', 'botocore', 'amazon-textract-caller>=0.0.11', 'Pillow>=9.2.*', 'PyPDF2>=2.5.*']
+requirements = ['boto3', 'botocore', 'amazon-textract-caller>=0.0.11', 'Pillow>=9.2.*', 'pypdf>=2.5.*']
```
Comment (schadem) [inline overlayer/setup.py:10]: Why change PyPDF2?

File: overlayer/textractoverlayer/image_tools.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
     sys.path.append('/opt')
 
 from PIL import Image
-from PyPDF2 import PdfReader
+from pypdf import PdfReader
```
Comment (schadem) [inline overlayer/textractoverlayer/image_tools.py:12]: same here and all below

File: prettyprinter/setup.py [hunk 5]
```diff
@@ -7,7 +7,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-requirements = ['boto3', 'botocore', 'amazon-textract-response-parser>=0.1.27', 'tabulate==0.8.10']
+requirements = ['boto3', 'botocore', 'amazon-textract-response-parser>=0.1.27', 'tabulate==0.9.*']
```
Comment (schadem) [inline prettyprinter/setup.py:10]: this one is ok, but should be it's own PR imho

File: setup.cfg [hunk 6]
```diff
@@ -1,2 +1,2 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+description_file = README.md
```
Comment (schadem) [inline setup.cfg:2]: guess this should be: 
`long_description = file:`

**Review by schadem — CHANGES_REQUESTED** [review]
some formatting, not sure why we want to change release.yml or update to the pypdf instead of PyPDF2.
Tabulate update is good, but can be its own PR.

File: helper/setup.py (hunk 1 above)
Comment (tb102122) [reply helper/setup.py:55]: since the version will be depreciated https://pypi.org/project/PyPDF2/

NOTE: The PyPDF2 project is going back to its roots. PyPDF2==3.0.X will be the last version of PyPDF2. Development will continue with [pypdf==3.1.0](https://pypi.org/project/pyPdf/).

File: overlayer/setup.py (hunk 3 above)
Comment (tb102122) [reply overlayer/setup.py:10]: since the version will be depreciated https://pypi.org/project/PyPDF2/

NOTE: The PyPDF2 project is going back to its roots. PyPDF2==3.0.X will be the last version of PyPDF2. Development will continue with [pypdf==3.1.0](https://pypi.org/project/pyPdf/).

File: overlayer/textractoverlayer/image_tools.py (hunk 4 above)
Comment (tb102122) [reply overlayer/textractoverlayer/image_tools.py:12]: see above

File: .github/workflows/release.yml (hunk 2 above)
Comment (tb102122) [reply .github/workflows/release.yml:22]: In order to resolve the warnings you have in the actions.
https://github.com/aws-samples/amazon-textract-textractor/actions/runs/3915318666

File: setup.cfg (hunk 6 above)
Comment (tb102122) [reply setup.cfg]: yes changed

File: helper/setup.py (hunk 1 above)
Comment (schadem) [reply helper/setup.py:55]: That is a good reason, @tb102122  :-)
Thx!

File: prettyprinter/setup.py (hunk 5 above)
Comment (schadem) [reply prettyprinter/setup.py:10]: I'll leave it in.

File: setup.cfg [hunk 7]
```diff
@@ -1,2 +1,2 @@
 [metadata]
-description_file = README.md
\ No newline at end of file
+long_description = file:
```
Comment (schadem) [inline setup.cfg:2]: Sorry, was not clear. Should be:
`long_description = file:README.md`

**Review by schadem — COMMENTED** [review]
Makes sense, just the long_description change

File: setup.cfg (hunk 7 above)
Comment (schadem) [reply setup.cfg:2]: Nevermind, I'll fix that quickly after merging

**Review by schadem — APPROVED** [review]
